.github/workflows/test-macos.yml

```diff
@@ -40,9 +40,7 @@ jobs:
     if: github.event.pull_request.draft == false
     strategy:
       fail-fast: false
```

Comment:
nit: I think this doesn't serve any purpose, this could be removed

Reply:
Removed